pkg/install/apiserver.yaml

```diff
@@ -67,6 +67,9 @@ spec:
           securityContext:
             runAsUser: 1000
       serviceAccountName: acorn-system
+      tolerations:
```

Comment:
Shouldn't we have a `Toleration` to allow for the control-plane taint and then an `Affinity` to attract the pods to those nodes?

Reply:
Great question Tyler. The caveat is that for hub every node is tainted so acorn controlplane will always go to the controlplane node. 

But for open-source, the taint is not guaranteed so it can go to any node as long as it can toleration the taints.

Reply:
Ah, understood. Thanks!